package/com.unity.formats.usd/Runtime/Scripts/IO/Scene/ImportHelpers.cs

```diff
@@ -8,7 +9,9 @@
 using UnityEngine;
 using USD.NET;
 using USD.NET.Unity;
+using Debug = UnityEngine.Debug;
```

Comment:
Not used?

Reply:
In this case I've swapped the `using System.Diagnostics;` for `using Stopwatch = System.Diagnostics.Stopwatch;` for consistency and removed the `using Debug = UnityEngine.Debug;` - is that better?